src/components/_banner.scss

```diff
@@ -0,0 +1,65 @@
+@use "../design-tokens/color";
+@use "../design-tokens/typography";
+
+@mixin BannerTheme($color: color.$color-utilitiesGray, $iconColor: color.$color-blue) {
+  border-left: 0;
+  
+  &::before {
+    position: absolute;
+    top: -1px;
+    left: 0;
+    width: 3px;
+    height: 100%;
+    background-color: $color;
+    border: 1px solid $color;
+    border-radius: 4px 0 0 4px;
+    content: '';
+  }
+
+  .Manifold__BannerIcon {
+    svg {
+      fill: $iconColor;
+    }
+  }
+}
```

Comment:
Nice! Love this

Reply:
Ah shoot. Just realized that we have a `@mixin Manifold__BannerIcon` and this is a CSS class. That may not work.

What if we just moved all this to the main `Manifold__Banner` styles? And manually just specified the SVG and border colors on the `--*` modifiers below? That would probably be more flexible if they didn’t use that exact CSS class.